packaging/deb/postinst

```diff
@@ -1,3 +1,13 @@
 #!/usr/bin/env sh
 echo "This software may collect information about you and your use of the software, and send that to Microsoft."
 echo "Please visit http://aka.ms/dotnet-cli-eula for more information."
+
+# Run 'dotnet new' to trigger the first time experience to initialize the cache
+echo ."NET Core Tools Telemetry
+
+The .NET Core Tools include a telemetry feature (https://github.com/dotnet/cli/pull/2145) that collects usage information. It’s important that the .NET Team understands how the tools are being used so that we can improve them.
```

Comment:
I don't think we should ever link to a PR from the product. It should be an aka.ms or fwlink and it should land in docs not github. 

The notices should be the same across platforms too.